sdk/keyvault/keyvault-keys/recordings/browsers/keys_client__create_read_update_and_delete_operations/recording_before_all_hook.json

```diff
@@ -0,0 +1,1985 @@
+{
+ "recordings": [
+  {
+   "method": "POST",
+   "url": "https://login.microsoftonline.com/azure_tenant_id/oauth2/v2.0/token",
+   "query": {},
+   "requestBody": "response_type=token&grant_type=client_credentials&client_id=azure_client_id&client_secret=azure_client_secret&scope=https%3A%2F%2Fvault.azure.net%2F.default",
+   "status": 200,
+   "response": "{\"token_type\":\"Bearer\",\"expires_in\":3600,\"ext_expires_in\":3600,\"access_token\":\"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6InU0T2ZORlBId0VCb3NIanRyYXVPYlY4NExuWSIsImtpZCI6InU0T2ZORlBId0VCb3NIanRyYXVPYlY4NExuWSJ9.eyJhdWQiOiJodHRwczovL3ZhdWx0LmF6dXJlLm5ldCIsImlzcyI6Imh0dHBzOi8vc3RzLndpbmRvd3MubmV0LzcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0Ny8iLCJpYXQiOjE1NjMzOTI0MTIsIm5iZiI6MTU2MzM5MjQxMiwiZXhwIjoxNTYzMzk2MzEyLCJhaW8iOiI0MkZnWUhnaVhOUjE3UE9UY09FVFpRS1hJcmsyQVFBPSIsImFwcGlkIjoiNTQyZWQ2MGItNjVhYy00MjY5LThiYjYtYTdlNjY4YzAyMDRmIiwiYXBwaWRhY3IiOiIxIiwiaWRwIjoiaHR0cHM6Ly9zdHMud2luZG93cy5uZXQvNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3LyIsIm9pZCI6IjdhYmY0YzA5LWQ4MTQtNDg0ZS05NjlhLTBkN2RmZTUxMjk1MCIsInN1YiI6IjdhYmY0YzA5LWQ4MTQtNDg0ZS05NjlhLTBkN2RmZTUxMjk1MCIsInRpZCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsInV0aSI6ImZuTTIzWjRzZVVDNS16bjQ4TnJ0QUEiLCJ2ZXIiOiIxLjAifQ.HqUESFxR5gZIKsJydHMAerOpxryaIQzkfGRglMJynE8xex0hcb8ddDXzzY3n2-0fxHAATAi3SvO-vg81qm6cmlGbaLpSsQI1sjlWVTcSU4cEP7PSUiJFhvNUwCLykjHS1QR3YhEkeyRC3ryXjdv46twsufd6ingkyWUT9f0IN9ItTODfAdz6z6lEpSkLPaJkybiuz9atwE9eHEbxvL2moFgtqSQsn4Rwu4FE8Qf-wbID8BUx7V7tvqw6YDuQyd7zF_lj8XiYkZMpu4D_kAEk8yr29Jg9LLAsI55PRkuUwTZB879WobDcdymGtqMUxmpvEIWdF8-ffHWJFTa-NGiVZg\"}",
```

Comment:
Is this `access_token` okay to be exposed ?

Reply:
I'm fixing it!